server/data/quotes.json

```diff
@@ -43,6 +43,10 @@
     "author": "Arab Proverb",
     "sentence": "Learn a language, and you’ll avoid a war."
   },
+  {
+  "author": "Arabic Proverb",
+  "sentence": "The strength of a person is in his intelligence and his tongue."
```

Comment:
Hi! Thanks for the PR but it's not aligned to the [contribution guidelines](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist). We need quotes regarding "learning a foreign language". Please update the quote accordingly.

Reply:
ok, noted.